public/scripts/suggestion.js

```diff
@@ -0,0 +1,17 @@
+$( document ).ready(function() {
+ $('#git_url').editableSelect();
+ $('#git_url').keyup(() => {
+   let q = $("#git_url").val()
```

Comment:
no need to change everywhere. Changing in this file would be enough,

> q for query and it is widely used everywhere

if that is the case, then no need to change.